system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/systemMetrics/common/Metrics.js

```diff
@@ -53,6 +57,13 @@ const styles = {
     },
     toolTipHead: {
         fontWeight: 600
+    },
+    info: {
+        color: "#999",
+        marginTop: 8,
+        marginRight: 27,
+        fontSize: 18
+
```

Comment:
unwanted new line

Reply:
Removed in developement repo https://github.com/wso2-cellery/mesh-observability/blob/master/components/global/portal/io.cellery.observability.ui/src/components/cells/metricsGraphs/index.js